docs/decisions/0032-stripe-price-lookup-keys.rst

```diff
@@ -0,0 +1,72 @@
+0032. Transition from Stripe Price IDs to Lookup Keys
+====================================================
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+Previously, the enterprise-access service used Stripe Price IDs for self-service purchasing (SSP) configuration and validation. Price IDs are generated by Stripe and change between environments (test, production), making configuration management difficult and error-prone.
```

Comment:
The key concern is less so that they change between environments, more so that they change every time somebody changes the price or any metadata about the price (e.g. frequency, name, etc.)